internal/tiltfile/docker_compose.go

```diff
@@ -37,42 +37,57 @@ type dcResourceSet struct {
 func (dc dcResourceSet) Empty() bool { return reflect.DeepEqual(dc, dcResourceSet{}) }
 
 func (s *tiltfileState) dockerCompose(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
-	configPaths := value.NewLocalPathListUnpacker(thread)
+	var configPaths starlark.Value
 
 	err := s.unpackArgs(fn.Name(), args, kwargs, "configPaths", &configPaths)
 	if err != nil {
 		return nil, err
 	}
 
-	for _, v := range configPaths.Value {
-		err = io.RecordReadPath(thread, io.WatchFileOnly, v)
-		if err != nil {
-			return nil, err
-		}
+	paths := starlarkValueOrSequenceToSlice(configPaths)
+
+	if len(paths) == 0 {
+		return nil, fmt.Errorf("Nothing to compose")
 	}
 
 	dc := s.dc
+	project := model.DockerComposeProject{ConfigPaths: dc.configPaths, YAML: dc.Project.YAML, ProjectPath: dc.Project.ProjectPath}
+
+	for _, val := range paths {
+		switch v := val.(type) {
+		case nil:
+			continue
+		case io.Blob:
+			project.YAML = v.String()
```

Comment:
is it intended that this should wipe out previous yaml? Or can you pass in multiple yaml files separate by "\n---\n" like you can with kubernetes?

Reply:
Just incomplete implementation at this point, I was waiting to decide whether to combine with previous YAML or generate an error.

That's an interesting idea about a YAML stream. Not sure if DC supports it but I'll check.